drivers/espi/espi_it8xxx2.c

```diff
@@ -517,7 +517,11 @@ static void port80_it8xxx2_isr(const struct device *dev)
 		ESPI_PERIPHERAL_NODATA
 	};
 
+#ifdef CONFIG_ESPI_IT8XXX2_PORT_81_CYCLE
+	evt.evt_data = gctrl->GCTRL_P80HDR | (gctrl->GCTRL_P81HDR << 8);
+#else
 	evt.evt_data = gctrl->GCTRL_P80HDR;
+#endif
```

Comment:
Use `if (IS_ENABLED(CONFIG_ESPI_IT8XXX2_PORT_81_CYCLE))` here instead of an ifdef.

Reply:
Done.